client/state/activity-log/log/schema.js

```diff
@@ -32,10 +32,8 @@ const activityItemsSchema = {
 				activityIsDiscarded: { type: 'boolean' },
 				activityIsRewindable: { type: 'boolean' },
 				activityName: { type: 'string' },
-				activityStatus: {
-					oneOf: [ { type: 'string' }, { type: 'null' } ],
-				},
-				activityTargetTs: { type: 'number' },
+				activityStatus: { type: [ 'null', 'string' ] },
```

Comment:
@dmsnell Worth doing something similar with `activityStatus` and `rewindId`? Based on the schema they're in the same boat 🙂